lib/beaker/dsl/helpers/host_helpers.rb

```diff
@@ -59,8 +60,8 @@ module HostHelpers
         #   on :dashboard, "echo hello"
         # @return [Result]   An object representing the outcome of *command*.
         # @raise  [FailTest] Raises an exception if *command* obviously fails.
-        def on(host, command, opts = {}, &block)
-          block_on host do | host |
+        def on(host, command, opts = {}, run_in_parallel=false, &block)
```

Comment:
I think we should put this in the `opts` rather than introduce a new parameter.  A nice benefit of that is that we no longer have to make a breaking change to this method.  And if we ever remove the `run_in_parellel` option, similarly, we won't have to make a breaking change to this method (or any method) signature.